gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -1103,31 +1104,47 @@ private HttpResponse handleExecuteMediaException(IOException e) throws IOExcepti
     throw new IOException(msg, e);
   }
 
-  private Storage.Objects.Get createDataRequest(String rangeHeader) throws IOException {
-    Storage.Objects.Get getObject = createRequest();
+  private Get createDataRequest(String rangeHeader) throws IOException {
```

Comment:
Please revert import and use more descriptive `Storage.Objects.Get` type, instead less obvious `Get`